erroranalysis/tests/test_error_report.py

```diff
@@ -49,6 +50,18 @@ def test_error_report_boston(self):
             run_error_analyzer(model, X_test, y_test, feature_names,
                                categorical_features)
 
+    def test_error_report_boston_pandas(self):
+        X_train, X_test, y_train, y_test, feature_names = \
+            create_boston_data()
+        X_train = pd.DataFrame(X_train, columns=feature_names)
+        X_test = pd.DataFrame(X_test, columns=feature_names)
```

Comment:
Can you move lines 65 and 57 into file common_utils.py?

Reply:
done!